runtime/src/rpc.rs

```diff
@@ -265,6 +282,12 @@ impl PolkaBtcProvider {
             .await?;
         Ok(())
     }
+
+    /// Read from latest block, even if it is not finalized yet
+    #[cfg(feature = "testing-utils")]
+    pub(crate) fn relax_storage_finalization_requirement(&mut self) {
```

Comment:
What's the reasoning for this?

Reply:
Integration tests fail with this, see discord